pkgs/by-name/pa/parabolic/package.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+, dotnetCorePackages
+, gtk4
+, libadwaita
+, pkg-config
+, wrapGAppsHook4
+, glib
+, shared-mime-info
+, gdk-pixbuf
+, blueprint-compiler
+, python3
+, ffmpeg
+}:
+
+buildDotnetModule rec {
+  pname = "parabolic";
+  version = "2023.12.0";
+
+  src = fetchFromGitHub {
+    owner = "NickvisionApps";
+    repo = "Parabolic";
+    rev = version;
+    hash = "sha256-mbGByw/wgovo81l2LDtDE5p+Mh6aJ5DOcZCNzVfmAtA=";
+    fetchSubmodules = true;
+  };
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+  pythonEnv = python3.withPackages(ps: with ps; [ yt-dlp ]);
+
+  projectFile = "NickvisionTubeConverter.GNOME/NickvisionTubeConverter.GNOME.csproj";
+  nugetDeps = ./deps.nix;
+  executables = "NickvisionTubeConverter.GNOME";
+
+   nativeBuildInputs = [
+    pkg-config
+    wrapGAppsHook4
+    glib
+    shared-mime-info
+    gdk-pixbuf
+    blueprint-compiler
+  ];
+
+  buildInputs = [ gtk4 libadwaita ];
+
+  runtimeDeps = [
+    gtk4
+    libadwaita
+    glib
+    gdk-pixbuf
+  ];
+
+  postInstall = ''
+    substituteInPlace NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.desktop.in --replace '@EXEC@' "NickvisionTubeConverter.GNOME"
```

Comment:
Usually we put source modifications under `postPatch`.

Can you please verify if it can be done in this particular case?

Reply:
Yes, seems to work the same as before